core/app/components/PubCRUD/actions.ts

```diff
@@ -154,11 +147,8 @@ export const _upsertPubValues = async ({
 		db
 			.insertInto("pub_values")
 			.values((eb) => {
-				return Object.entries(fields).map(([slug, value]) => {
+				return Object.entries(values).map(([slug, value]) => {
 					return {
-						id:
-							pubFields.find((pf) => pf.slug === slug && pf.pubId === pubId)
-								?.pubValueId ?? undefined,
```

Comment:
We no longer need to pass the id here because we don't rely on the id to detect if there's an existing value. Now we use the unique constraint in the `on conflict` clause below